### ISSUE TITLE
mySql function conversion fails with error

### ISSUE THREAD

**Issue by mrechte** [darold/ora2pg#746]
Hello,
Trying to export mySql functions, I get the following error:
```
$ ora2pg -t FUNCTION -c ora2pg.conf
DBD::mysql::st execute failed: Unknown column 'DATA_TYPE' in 'field list' at /var/lib/pgsql/perl5/lib/perl5/Ora2Pg/MySQL.pm line 693.
DBD::mysql::st execute failed: Unknown column 'DATA_TYPE' in 'field list' at /var/lib/pgsql/perl5/lib/perl5/Ora2Pg/MySQL.pm line 693.
```
Thanks

**Comment by darold** [comment]
What is your mysql version?

**Comment by mrechte** [comment]
```
$ rpm -q mysql
mysql-5.1.52-1.el6_0.1.x86_64
```

**Comment by darold** [comment]
Commit 2adeb67 fixes this issue, thanks for the report.

**Comment by mrechte** [comment]
Hello,
Thanks for fixing.
I still have the issue:
```
$ ~/usr/local/bin/ora2pg --version
Ora2Pg v20.0
$ ~/usr/local/bin/ora2pg -t SHOW_VERSION -c ~/etc/ora2pg/ora2pg-cdp.conf                                          
5.1.52
$ ~/usr/local/bin/ora2pg -t FUNCTION -c ~/etc/ora2pg/ora2pg-cdp.conf >mysql2pg_cdp_function.sql
DBD::mysql::st execute failed: Unknown column 'DATA_TYPE' in 'field list' at /home/cdp/usr/local/share/perl5/Ora2Pg/MySQL.pm line 693.
DBD::mysql::st execute failed: Unknown column 'DATA_TYPE' in 'field list' at /home/cdp/usr/local/share/perl5/Ora2Pg/MySQL.pm line 693.
```

**Comment by darold** [comment]
Hi,

Yes unfortunately it was too late to apply the patches on MySQL migration in version 20.0, you have to use latest development code. The fixes will be available in version 20.1.

Regards,
